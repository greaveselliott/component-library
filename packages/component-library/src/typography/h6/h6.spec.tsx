import React from 'react';
import { shallow } from 'enzyme';

import H6 from './h6.component';

test('Heading one should render correctly', () =>
  expect(shallow(<H6>abc</H6>)).toMatchSnapshot());
