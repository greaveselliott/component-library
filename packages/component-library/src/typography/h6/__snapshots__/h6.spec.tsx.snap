// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading one should render correctly 1`] = `
<h6
  className="heading-six"
>
  abc
</h6>
`;
